blocks/redirects-formatter/redirects-formatter.js

```diff
@@ -45,31 +45,37 @@ function handleError(e, eSection) {
   eSection.classList.add('error-border');
 }
 
-export function generateRedirectList(urls, locales, handler) {
+export function generateRedirectList(urls, locales) {
   const inputSection = document.querySelector('.redirects-text-area');
   const checkboxSection = document.querySelector('.checkbox-container');
   const errorMessage = 'Invalid URL. URLs must start with "https://" e.g: "https://business.adobe.com"';
 
   return urls.reduce((rdx, urlPair) => {
-    if (!locales.length) handler(NO_LOCALE_ERROR, checkboxSection);
+    if (!locales.length) handleError(NO_LOCALE_ERROR, checkboxSection);
 
     locales.forEach((locale) => {
       let from;
       let to;
       try {
         from = new URL(urlPair[0]);
       } catch (e) {
-        handler(errorMessage, inputSection);
+        handleError(errorMessage, inputSection);
         return;
       }
       try {
         to = new URL(urlPair[1]);
       } catch (e) {
-        handler(errorMessage, inputSection);
+        handleError(errorMessage, inputSection);
         return;
       }
       const fromPath = from.pathname.split('.html')[0];
-      rdx.push([`/${locale}${fromPath}`, `${to.origin}/${locale}${to.pathname}`]);
+      const toPath = () => {
+        if (to.origin !== 'https://business.adobe.com' || to.pathname.includes('blog')) {
+          return to.pathname;
+        }
+        return `${to.pathname}.html`;
```

Comment:
What about the homepage?

Also, what about www.adobe.com pages that should have `.html`? Maybe open up the check to origins that end in `.adobe.com`?

Reply:
Nice! Good points both. Thank you!